model/flow/address_test.go

```diff
@@ -15,6 +15,27 @@ type addressWrapper struct {
 	Address Address
 }
 
+func TestHexToAddress(t *testing.T) {
+
+	type testCase struct {
+		literal string
+		value   []byte
+	}
+
+	for _, test := range []testCase{
+		{"123", []byte{0x1, 0x23}},
+		{"1", []byte{0x1}},
+		// leading zero
+		{"01", []byte{0x1}},
```

Comment:
Should we add `0001`? I do think i've seen an address that started with three zeros though https://flowscan.org/account/0x0001dfd237612b1e